test/parallel/test-http-set-trailers.js

```diff
@@ -78,7 +78,7 @@ server.on('listening', function() {
   });
 
   c.on('data', function(chunk) {
-    //console.log(chunk);
+    // console.log(chunk);
```

Comment:
Suggestion: remove this. If the test fails, this has to be debugged properly anyway and maybe needs a code change but the comment does not help in this case.